vic/extensions/rout_stub/src/rout.c

```diff
@@ -32,13 +32,6 @@
 void
 rout_alloc(void)
 {
-    extern rout_struct rout;
-
-    // We need to write at least something to the VIC state file (if enabled).
-    // Therefore we make the dimension size 1*1.
-    rout.rout_param.n_outlets = 1;
-    rout.rout_param.full_time_length = 1;
-    rout.ring[1] = 0;
```

Comment:
If I'm understanding correctly - if we are referencing those values, we wouldn't be using this version of `rout.c` because this is `rout_stub`, meaning VIC without routing.

Reply:
True, that is how it should be but I wan't to make sure the state file function is treating this properly. @wietsefranssen should be able to help us understand why this was written in this way.